### ISSUE TITLE
Inferno SMART auth fails during CORS token redemption with 400 Error

### ISSUE THREAD

**Issue by muthink** [Azure-Samples/azure-health-data-services-samples#43]
As per the deployment.md document in samples\Patient and Population Services G10, a web and not SPA app was created, yet we are getting an AADSTS9002326 Error, which states only SPA authentication permits such use.

Note that manifest.json files are not found, yielding some 404 errors, which do not seem to alter the workflow. Why is code, in the sample, checking for manifest.json files, when none are deployed? Not sure this is relevant.

Inferno reports a failure in step 1.305 of the ONC Certification (g)(10) Standardized API
**1.3.05 OAuth token exchange request succeeds when supplied correct information**
_Unexpected response status: expected 200, but received 400_

GET from https://inferno.healthit.gov/suites/custom/smart/redirect?code=[Redacted], with referer=https://login.microsoftonline.com/

POST fails with 400 error:
```
https://azurefhirdev-apim.azure-api.net/smart/token
code=[Redacted]&grant_type=authorization_code&redirect_uri=https%3A%2F%2Finferno.healthit.gov%2Fsuites%2Fcustom%2Fsmart%2Fredirect
```

Inferno's response:
```
{
  "error": "invalid_request",
  "error_description": "AADSTS9002326: Cross-origin token redemption is permitted only for the 'Single-Page Application' client-type. Request origin: 'http://localhost'.\r\nTrace ID: 8b8f045e-93cb-439e-9b9e-95c745b4e500\r\nCorrelation ID: 57a70950-8fc3-4a48-9f2d-cf6037567534\r\nTimestamp: 2023-02-22 17:02:05Z",
  "error_codes": [
    9002326
  ],
  "timestamp": "2023-02-22 17:02:05Z",
  "trace_id": "8b8f045e-93cb-439e-9b9e-95c745b4e500",
  "correlation_id": "57a70950-8fc3-4a48-9f2d-cf6037567534",
  "error_uri": "https://login.microsoftonline.com/error?code=9002326"
}
```
NOTE, before this point, a 404 error is encountered when trying to load the manifest.json from the registered static app: https://[redacted].azurestaticapps.net/manifest.json

**Comment by EXPEkesheth** [comment]
Thanks for reporting the issue ! We will investigate and respond back with details, as we identify.

**Comment by mikaelweave** [comment]
@EXPEkesheth - this is something I can investigate. I have a good idea where the issue is.

**Comment by muthink** [comment]
I'm very close to completing the Inferno g10 test part, 1.3. using your sample. This one item remains. I'm watching this closely and ready to retest the moment you release the fix.

**Comment by EXPEkesheth** [comment]
Thanks @mikaelweave . @muthink - please let us know if you are unblocked and this issue can be closed.

**Comment by muthink** [comment]
I updated with the latest from main and got further in the inferno authorization sequence, but another error occurred, and I'll open a new ticket for that. I'll also check permissions of the app, etc...
![image](https://user-images.githubusercontent.com/1256768/224323347-2f0e737d-31e2-44e7-b267-d8da695877ee.png)
